diff --git a/chrome/common/media/cdm_host_file_path.cc b/chrome/common/media/cdm_host_file_path.cc
index c6030f417db0ea1aca7f1a618a174ab6bc9cb516..f695f1d564b43374724c71e08b877aff41cf16c0 100644
--- a/chrome/common/media/cdm_host_file_path.cc
+++ b/chrome/common/media/cdm_host_file_path.cc
@@ -17,7 +17,7 @@
 #include "chrome/common/chrome_constants.h"
 #endif
 
-#if defined(GOOGLE_CHROME_BUILD)
+#if defined(GOOGLE_CHROME_BUILD) || defined(BRAVE_CHROMIUM_BUILD)
 
 namespace {
 
@@ -42,8 +42,10 @@ void AddCdmHostFilePaths(
 
 #if defined(OS_WIN)
 
+  LOG(ERROR) << "BSC]] AddCdmHostFilePaths 1";
+
   static const base::FilePath::CharType* const kUnversionedFiles[] = {
-      FILE_PATH_LITERAL("chrome.exe")};
+      FILE_PATH_LITERAL("brave.exe")};
   static const base::FilePath::CharType* const kVersionedFiles[] = {
       FILE_PATH_LITERAL("chrome.dll"), FILE_PATH_LITERAL("chrome_child.dll")};
 
@@ -53,6 +55,8 @@ void AddCdmHostFilePaths(
     NOTREACHED();
   base::FilePath version_dir(chrome_exe_dir.AppendASCII(CHROME_VERSION_STRING));
 
+  LOG(ERROR) << "BSC]] AddCdmHostFilePaths 2: version folder: " << version_dir.value();
+
   cdm_host_file_paths->reserve(arraysize(kUnversionedFiles) +
                                arraysize(kVersionedFiles));
 
@@ -61,14 +65,14 @@ void AddCdmHostFilePaths(
     base::FilePath file_path = chrome_exe_dir.Append(kUnversionedFiles[i]);
     base::FilePath sig_path =
         GetSigFilePath(version_dir.Append(kUnversionedFiles[i]));
-    DVLOG(2) << __func__ << ": unversioned file " << i << " at "
+    LOG(ERROR) << "BSC]] AddCdmHostFilePaths 3: unversioned file " << i << " at "
              << file_path.value() << ", signature file " << sig_path.value();
     cdm_host_file_paths->emplace_back(file_path, sig_path);
   }
 
   for (size_t i = 0; i < arraysize(kVersionedFiles); ++i) {
     base::FilePath file_path = version_dir.Append(kVersionedFiles[i]);
-    DVLOG(2) << __func__ << ": versioned file " << i << " at "
+    LOG(ERROR) << "BSC]] AddCdmHostFilePaths 4: versioned file " << i << " at "
              << file_path.value();
     cdm_host_file_paths->emplace_back(file_path, GetSigFilePath(file_path));
   }
